edx_proctoring/migrations/0014_update_is_attempt_active.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+
+import logging
+
+from django.db import migrations
+
+def update_is_attempt_active(apps, schema_editor):
+
+    def _check_and_update_field(review_object):
+        # helper function that will check to see if an active attempt exists. If it doesn't
+        # the is_attempt_active field will be updated
+        ProctoredExamAttempt = apps.get_model('edx_proctoring', 'ProctoredExamStudentAttempt')
+        try:
+            ProctoredExamAttempt.objects.get(attempt_code=review_object.attempt_code)
+        except ProctoredExamAttempt.DoesNotExist:
+            review_object.is_attempt_active = False
+            review_object.save()
+
+    ProctoredExamReview = apps.get_model('edx_proctoring', 'ProctoredExamSoftwareSecureReview')
+    ProctoredExamReviewHistory = apps.get_model('edx_proctoring', 'ProctoredExamSoftwareSecureReviewHistory')
+
+    for review in ProctoredExamReview.objects.filter(is_attempt_active=True):
```

Comment:
This migration might run pretty long time. Could you first get to know how many review we will be iterating over? Also, please print out each Review record changed so we know that the migration is moving along successfully?